Source/RageCommunity.Library/Extensions/PedExtensions.cs

```diff
@@ -23,14 +23,51 @@ public static class PedExtensions
         public static void ShootFromVehicle(this Ped ped, Ped target, float unknown = 0) => NativeWrappers.TaskVehicleShootAtPed(ped, target, unknown);
 
         /// <summary>
-        /// Causes this ped to start playing the specified scenario.
+        /// Causes this ped to start playing the specified scenario in place.
         /// </summary>
-        public static void StartScenarioInPlace(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenario, unkDelay, playEnterAnimation);
-
+        public static void StartScenario(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
```

Comment:
Why was this method name changed?

Reply:
Because I added an overload to play scenario at the specified position. but I'm fine if you change it back to the original

Reply:
I see the other method, and suggested a change to that name as well for the sake of clarity and consistency with our naming conventions.  Thanks!